### ISSUE TITLE
demo？

### ISSUE THREAD

**Issue by lunalulu** [huguyuehuhu/HCN-pytorch#14]
I implemented a skeleton extraction, and then motion recognition demo；
I modified the use of a smaller number of key points （14）for motion recognition, retraining，there are 6 classes。The training accuracy was 0.90102261。
also I modified the network parameters, window size is 16, person num is 1。
But demo's action recognition is poor.
I accumulated 16 frames of input into the HCN, the next time from the previous accumulation of half the beginning of the accumulation。

Please give me some advice, thank you

**Comment by huguyuehuhu** [comment]
Maybe you should check if your model is overfiting , beacause the training accuracy is seems good.  It really hard to provide any useful suggestion without any specific information about training /testing curves.

**Comment by lunalulu** [comment]
@huguyuehuhu OK，thanks
another problem：
When I only take the data of the previous 49 classes as the training set and test set, the training effect is very poor，using the two-dimensional coordinates of the skeleton, of course. However, it is good to select 9 types of training results, although it is possible to overfit.
`
188 2019-04-26 14:20:59,997:INFO: Epoch 30/600
189 2019-04-26 14:21:08,408:INFO: - Train metrics: accuracytop5: 0.096 ; loss: 3.892 ; ls_CE: 3.892 ; ls_all: 3.892 ; accuracytop1: 0.018
190 2019-04-26 14:21:10,102:INFO: - Eval metrics : accuracytop5: 0.102 ; loss: 3.892 ; ls_CE: 3.892 ; ls_all: 3.892 ; accuracytop1: 0.020
191 2019-04-26 14:21:10,158:INFO: Epoch 31/600
192 2019-04-26 14:21:18,724:INFO: - Train metrics: accuracytop5: 0.095 ; loss: 3.893 ; ls_CE: 3.893 ; ls_all: 3.893 ; accuracytop1: 0.019
193 2019-04-26 14:21:20,352:INFO: - Eval metrics : accuracytop5: 0.103 ; loss: 3.892 ; ls_CE: 3.892 ; ls_all: 3.892 ; accuracytop1: 0.021
`

**Comment by huguyuehuhu** [comment]
1. You can see the training accuracy is about 1/49=0.02, that means your model learned nothing but random guess the results.  It is impossible, even a one layer LSTM could perform better than random guess, so it is surely that there is something wrong in your data preparation or somewhere else.
2. Maybe  you can, at least, try:
1) use `visdom` to see the accuracy and loss curves during all the training process, it obviously your loss did not decrease. NOT just focusing on the final accuracy at the terminal.
2) try to use one layer LSTM replace HCN model, once you can make your  loss decrease, then you can sure your code processes right. After that you can use to HCN to pursuit a better performance.
